R/tidy.R

```diff
@@ -5,6 +5,15 @@
 #' \code{\link{deparse}}. It can also replace \code{=} with \code{<-} where
 #' \code{=} means assignments, and reindent code by a specified number of spaces
 #' (default is 4).
+#'
+#' If the value of the argument \code{width.cutoff} is wrapped in
+#' \code{\link{I}()} (e.g., \code{I(60)}), it will be treated as the \emph{upper
+#' bound} on the line width, but this upper bound may not be satisfied. In this
+#' case, the function will perform a binary search for a width value that can
+#' make \code{deparse()} return code with line width smaller than or equal to
+#' the \code{width.cutoff} value. If the search fails to find such a value, it
+#' will emit a warning, which can be suppressed by the global option
+#' \code{options(formatR.width.warning = FALSE)}.
```

Comment:
Thanks for merging this PR! It will be great to have this Just Work.

I think this paragraph isn't 100% clear, because it can be read as the upper bound potentially not being satisfied *after* `I()` is used. Suggested edit:

```r
#' If the value of the argument \code{width.cutoff} is wrapped in
#' \code{\link{I}()} (e.g., \code{I(60)}), it will be treated as the
#' \emph{upper bound} on the line width. The corresponding argument to
#' \code{deparse()} is actually a lower bound, and so the function
#' will perform a binary search for a width value that can make
#' \code{deparse()} return code with line width smaller than or equal
#' to the \code{width.cutoff} value. If the search fails to find such
#' a value, it will emit a warning, which can be suppressed by
#' the global option \code{options(formatR.width.warning = FALSE)}.
```

Reply:
Thanks for the suggestion! I'll commit that later.